File: Easy/lc136.py
# Single Number
class Solution(object):
    def singleNumber(self, nums):
        for num in nums:
            if nums.count(num) == 1:
                return num
s=Solution()
print(s.singleNumber([2,2,1]))          # Output: 1
print(s.singleNumber([4,1,2,1,2]))      # Output: 4
print(s.singleNumber([1]))               # Output: 1    

File: Easy/lc136.java
class SingleNumberSolution {
    public int singleNumber(int[] nums) {
        int n=nums[0];
        for(int i = 1; i<nums.length;i++){
            n=n^nums[i];
        }
        return n;
    }
    public static void main(String[] args) {
        SingleNumberSolution sol = new SingleNumberSolution();
        int[] nums = {2, 2, 1};
        System.out.println(sol.singleNumber(nums)); // Output: 1
    }
}